testing/end_to_end.rst

```diff
@@ -799,6 +799,56 @@ variable to ``false`` in your style file:
 
     $enable-smooth-scroll: false;
 
+Assets not loading
```

Comment:
Maybe we can use a more specific title as this problem only concern PHP build-in server ?

Reply:
Yes you are right, its only when using built-in server.